go.mod

```diff
@@ -25,6 +24,6 @@ require (
 	// Once there is a 0.18 release, we can import a semver release.
 	k8s.io/kubectl v0.0.0-20191219154910-1528d4eea6dd
 	sigs.k8s.io/cli-utils v0.15.0
-	sigs.k8s.io/kustomize/cmd/config v0.3.0
-	sigs.k8s.io/kustomize/kyaml v0.3.1-0.20200618190311-fb6830c98a78
+	sigs.k8s.io/kustomize/cmd/config v0.0.0-20200629173832-441581b74521
```

Comment:
What is this version ? Did this seep in by mistake ?

Reply:
that's a new version I added by running go get sigs.k8s.io/kustomize/cmd/config of a specific sha with my delete setter change.